core/src/test/java/org/apache/iceberg/TestManifestReader.java

```diff
@@ -41,6 +44,12 @@ public static Object[] parameters() {
     return new Object[] {1, 2};
   }
 
+  private static final RecursiveComparisonConfiguration FILE_COMPARISON_CONFIG =
+      RecursiveComparisonConfiguration.builder()
+          .withIgnoredFields(
+              "dataSequenceNumber", "fileOrdinal", "fileSequenceNumber", "fromProjectionPos")
```

Comment:
So this will compare split offsets now?

Reply:
yes, it will compare all fields from the input data files, except for these excluded fields which aren't expected to match